solr/core/src/test/org/apache/solr/cloud/TestPullReplicaErrorHandling.java

```diff
@@ -118,7 +118,7 @@ public void tearDown() throws Exception {
       log.info("tearDown deleting collection");
       CollectionAdminRequest.deleteCollection(collectionName).process(cluster.getSolrClient());
       log.info("Collection deleted");
-      waitForDeletion(collectionName);
+      TestPullReplica.waitForDeletion(collectionName);
```

Comment:
This method is already invoked in `TestPullReplicaWithAuth` (no change in this PR). Since they are related tests, it makes sense to shared it instead of duplicating the code.